server/index.js

```diff
@@ -37,6 +40,31 @@ app.get('/products/:product_id', (req, res) => {
     });
 });
 
+app.get('/reviews', (req, res) => {
+  const { query } = url.parse(req.url);
+  const id = query.slice(query.indexOf('=') + 1);
+  axios.get(`${API_BASE_URL}/reviews?product_id=${id}`, options)
+    .then((response) => {
+      res.send(response.data);
+    })
+    .catch((err) => {
+      res.send(`Error:: ${err}`);
+    });
+});
+app.put('/reviews/:review_id/helpful', (req, res) => {
+  axios.put(`${API_BASE_URL}/reviews/${req.params.review_id}/helpful`, options)
+    .then(() => res.sendStatus(204))
+    .catch((err) => {
+      res.send(`Error:: ${err}`);
```

Comment:
This is just a string, not an actual error. I would maybe send an error status code like '404' or actually throw an Error which could be handled in the catch block.

Reply:
I will send a 404